arch/ARM/Nordic/drivers/nrf_common/nrf-gpio-tasks_and_events.adb

```diff
@@ -34,6 +34,31 @@ with HAL;              use HAL;
 
 package body nRF.GPIO.Tasks_And_Events is
 
+   -----------------------------------
+   -- Acknowledge_Channel_Interrupt --
+   -----------------------------------
+
+   procedure Acknowledge_Channel_Interrupt (Chan : GPIOTE_Channel) is
+   begin
+      GPIOTE_Periph.EVENTS_IN (Integer (Chan)) := 0;
+   end Acknowledge_Channel_Interrupt;
+
+   --------------------------------
+   -- Acknowledge_Port_Interrupt --
+   --------------------------------
+
+   procedure Acknowledge_Port_Interrupt is
+   begin
+      GPIOTE_Periph.EVENTS_PORT := 0;
+   end Acknowledge_Port_Interrupt;
+
+   --------------------------
+   -- Channel_Event_Is_Set --
+   --------------------------
+
+   function Channel_Event_Is_Set (Chan : GPIOTE_Channel) return Boolean is
```

Comment:
Not a huge deal but we usually don't add a `is` in the names of test functions like that.
Also we put the `is` keyword on a new line:
```ada
function Channel_Event_Set (Chan : GPIOTE_Channel) return Boolean
is (GPIOTE_Periph.EVENTS_IN (Integer (Chan)) /= 0);
```

Reply:
Added another commit to sort this.

After a bit more poking about, I see that I could have used `nRF.Events`. You might think that my effort could have been better spent adding how-to comments to that package?

By the way, the comments in nrf-events.adb mention the nRF series RM section 9; in nRF51_RM_v3.0, this is section 10.